src/context/talentLayer.tsx

```diff
@@ -60,13 +75,11 @@ const TalentLayerProvider = ({ children }: { children: ReactNode }) => {
 
       const currentUser = userResponse.data.data.users[0];
 
-      const platformResponse = await getPlatform(
-        chainId,
-        process.env.NEXT_PUBLIC_PLATFORM_ID || '',
-      );
-      const platform = platformResponse?.data?.data?.platform;
-      currentUser.isAdmin = platform?.address === currentUser?.address;
-
+      if (talentLayerClient) {
```

Comment:
you should not need this here, already got the check above

Reply:
ah
good catch